common/utils/Clock.cpp

```diff
@@ -251,12 +263,26 @@ const TimeStamp TimeStamp::operator-(const TimeInterval &interval) const {
   return TimeStamp(m_tv - interval.m_interval);
 }
 
-void Clock::CurrentTime(TimeStamp *timestamp) const {
+void Clock::CurrentMonotonicTime(TimeStamp *timestamp) const {
+#ifdef CLOCK_MONOTONIC
+  struct timespec ts;
+  clock_gettime(CLOCK_MONOTONIC, &ts);
```

Comment:
Can you add this to AC_CHECK_FUNCS in configure.ac so we ensure we've actually got clock_gettime available.

Reply:
Done.